packages/core/botpress/build.sh

```diff
@@ -2,8 +2,11 @@ echo "--> Cleaning up old build"
 rm -rf lib
 mkdir -p lib/cli/templates
 
-echo "--> Bundling app"
-node webpack.js --compile
+echo "--> Bundling server part"
+NODE_ENV=production node ./webpack.server.js --compile
+
+echo "--> Bundling web part"
+NODE_ENV=production node ./webpack.web.js --compile
```

Comment:
@dmytropanontko , could these be started also via `concurrently` so that they run in parallel?

Reply:
@epaminond I think yes